src/components/MenuButton.js

```diff
@@ -0,0 +1,16 @@
+const MenuButton = (textInsideButton, onButtonClick) => {
+  const button = document.createElement('button');
+  button.classList.add('menuButton');
+
+  const innerTextButton = document.createElement('span');
+  innerTextButton.classList.add('menuButton__text');
+  innerTextButton.innerText = textInsideButton;
+
+  button.appendChild(innerTextButton);
```

Comment:
Moim zdaniem ten span jest zbędny
```suggestion
  button.textContent  = textInsideButton;
```

W kolejnych komentarzach zamieszczę Ci zmiany z tym związane